pkg/cmd/start/bootstrap.go

```diff
@@ -230,7 +229,7 @@ func detectNamespacePermissions(ctx context.Context, mgr manager.Manager) {
 
 func setOperatorScope(ctx context.Context, namespace string) {
 	tracer := otel.GetTracerProvider().Tracer(v1.BootstrapTracer)
-	ctx, span := tracer.Start(ctx, "setOperatorScope")
+	ctx, span := tracer.Start(ctx, "setOperatorScope") // nolint:ineffassign,staticcheck
```

Comment:
The other way to make the linter happy would be to define a span var and then use `=` on this line. This way we don't have to disable linting. 

I would maybe prefer the version that passes linter, but it's not great either. I will leave it up to you.